--- config.m4	2012-05-30 20:21:12.000000000 +0000
+++ config.m4.patched	2013-06-10 19:24:10.641257714 +0000
@@ -52,8 +52,8 @@
   else	
     AC_MSG_CHECKING(for MagickWand.h header file)
 
-    if test -r $WAND_DIR/include/ImageMagick/wand/MagickWand.h; then
-      AC_MSG_RESULT(found in $WAND_DIR/include/ImageMagick/wand/MagickWand.h)
+    if test -r $WAND_DIR/include/ImageMagick-6/wand/MagickWand.h; then
+      AC_MSG_RESULT(found in $WAND_DIR/include/ImageMagick-6/wand/MagickWand.h)
     else
       AC_MSG_ERROR(Cannot locate header file MagickWand.h)
     fi
